src/test/regress/expected/failure_create_index_concurrently.out

```diff
@@ -0,0 +1,200 @@
+--
+-- failure_create_index_concurrently
+-- test create index concurrently command
+-- failure.
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+SET citus.shard_count = 4; -- two per worker
+CREATE SCHEMA index_schema;
+SET SEARCH_PATH=index_schema;
+CREATE TABLE index_test(id int, value_1 int, value_2 int);
+SELECT create_distributed_table('index_test', 'id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- kill the connection when create command is issued
+SELECT citus.mitmproxy('conn.onQuery(query="CREATE").kill()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+CREATE INDEX CONCURRENTLY idx_index_test ON index_test(id, value_1);
+ERROR:  CONCURRENTLY-enabled index command failed
+DETAIL:  CONCURRENTLY-enabled index commands can fail partially, leaving behind an INVALID index.
+HINT:  Use DROP INDEX CONCURRENTLY IF EXISTS to remove the invalid index, then retry the original command.
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+-- verify index is not created
+SELECT * FROM run_command_on_workers($$SELECT count(*) FROM pg_indexes WHERE indexname LIKE 'idx_index_test%' $$)
+WHERE nodeport = :worker_2_proxy_port;
```

Comment:
Another way would have been to write `WHERE nodeport != :worker_1_port`

Reply:
We could also change :worker_2_port and have it always point at the proxy, that sounds like a cleaner option.